.ci/deps.sh

```diff
@@ -67,8 +67,8 @@ fi
 
 deps_perl="perl libperl-critic-perl"
 
-sudo apt-get -y update
-sudo apt-get -y --no-install-recommends install $deps $deps_perl $deps_infer
+sudo apt-get -y update|grep -v '\%'
```

Comment:
@Techievena No, that wouldn't do anything bad to the log files, because they're basically "just output text, lol"